Tests/GitHubRepositories.tests.ps1

```diff
@@ -688,16 +850,167 @@ try
+        }
+
+        Context 'When getting GitHub Repository Contributors' {
+            BeforeAll {
+                $getGitHubRepositoryContributorParms = @{
+                    OwnerName = $repo.owner.login
+                    RepositoryName = $repoName
+                }
+                $contributors = Get-GitHubRepositoryContributor @getGitHubRepositoryContributorParms
+            }
+
+            It 'Should return objects of the correct type' {
+                $contributors.PSObject.TypeNames[0] | Should -Be 'GitHub.User'
+            }
+
+            It 'Should return at least one result' {
+                $contributors.count | Should -BeGreaterOrEqual 1
+            }
+
+            It "Should return the correct membership" {
+                $repo.owner.login | Should -BeIn $contributors.login
+            }
+        }
+
+        Context 'When getting Github Repository Contributors with Statistics' {
+            BeforeAll {
+                $getGitHubRepositoryContributorParms = @{
+                    OwnerName = $repo.owner.login
+                    RepositoryName = $repoName
+                    IncludeStatistics = $true
+                }
+                $contributors = Get-GitHubRepositoryContributor @getGitHubRepositoryContributorParms
+            }
+
+            It 'Should return objects of the correct type' {
+                $contributors.PSObject.TypeNames[0] | Should -Be 'GitHub.User'
+            }
+
+            It 'Should return at least one result' {
+                $contributors.count | Should -BeGreaterOrEqual 1
+            }
+
+            It 'Should return the correct membership' {
+                $repo.owner.login | Should -BeIn $contributors.author.login
+            }
+
+            It 'Should return the correct properties' {
+                $contributors.weeks | Should -Not -BeNullOrEmpty
+            }
+        }
+
+        Context 'When getting Github Repository Contributors including Anonymous' {
+            BeforeAll {
+                $getGitHubRepositoryContributorParms = @{
+                    OwnerName = $repo.owner.login
+                    RepositoryName = $repoName
+                    IncludeAnonymousContributors = $true
+                }
+                $contributors = Get-GitHubRepositoryContributor @getGitHubRepositoryContributorParms
+            }
+
+            It 'Should return objects of the correct type' {
+                $contributors.PSObject.TypeNames[0] | Should -Be 'GitHub.User'
+            }
+
+            It 'Should return at least one result' {
+                $contributors.count | Should -BeGreaterOrEqual 1
+            }
+
+            It 'Should return the correct membership' {
+                $repo.owner.login | Should -BeIn $contributors.login
+            }
+        }
+
+        AfterAll {
+            Remove-GitHubRepository -Uri $repo.svn_url -Confirm:$false
+        }
+    }
+
+    Describe 'GitHubRepositories\Get-GitHubRepositoryCollaborator' {
```

Comment:
Similar to above, there are some existing Collaborator tests that this appears to be duplicative of:

https://github.com/microsoft/PowerShellForGitHub/blob/2385b5cf5d959a7581bf968f15f346d9a0ff816b/Tests/GitHubRepositories.tests.ps1#L789-L815